ghcide/src/Development/IDE/Spans/Documentation.hs

```diff
@@ -72,16 +72,14 @@ getDocumentationsTryGhc env mod names = do
   res <- fun
   case res of
       Left _    -> return []
-      Right res -> zipWithM unwrap (fmap snd $ Map.toList res) names
+      Right res -> sequenceA $ unwrap <$> Map.toList res
   where
     fun :: IO (Either [FileDiagnostic] (Map.Map Name (Either String (Maybe HsDocString, Map.Map Int HsDocString))))
     fun = catchSrcErrors (hsc_dflags env) "docs" $ getDocsBatch env mod names
 
-    unwrap (Right (Just docs, _)) n = SpanDocString docs <$> getUris n
-    unwrap _ n                      = mkSpanDocText n
-
-    mkSpanDocText name =
-      SpanDocText [] <$> getUris name
```

Comment:
A simple function that is used right on that place once, so it is away below https://wiki.haskell.org/Fairbairn_threshold